.github/workflows/ami-release.yml

```diff
@@ -50,28 +50,24 @@ jobs:
           endpoint: builders
       - uses: docker/build-push-action@v5
         with:
-          load: true
           build-args: |
             ${{ steps.args.outputs.result }}
           target: extensions
           tags: supabase/postgres:extensions
           platforms: linux/${{ matrix.arch }}
+          outputs: type=tar,dest=/tmp/extensions.tar
           cache-from: type=gha,scope=${{ github.ref_name }}-latest-${{ matrix.arch }}
           # No need to export extensions cache because latest depends on it
       - name: Extract built packages
         run: |
-          mkdir -p /tmp/extensions ansible/files/extensions
-          docker save supabase/postgres:extensions | tar xv -C /tmp/extensions
```

Comment:
`docker save` starting exporting the `.tar` in a different format for some reason: https://github.com/supabase/postgres/actions/runs/7637264535